src/grip/GripHandle.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+
+import { color, space } from '../_utils/branding'
+
+export const GripHandle = styled.div`
+  & {
```

Comment:
It's a brand new component, you don't need to declare higher specificity, so you can remove the `&`

Reply:
Done ;)